DigitalProductInnovationAndDevelopment/Security-Compliance-Assessment
# Add unit tests

Add unit tests for important custom components, to be found in folders named `__tests__`.

Steps to run unit tests:
1. Make sure `"jsx": "react",` in `tsconfig.json`
2. In terminal, run `pnpm run test`

![image](https://github.com/user-attachments/assets/7c55367b-5b78-4f1e-8d9d-49f835012226)
